spec/views/hq/orphans/_form_spec.rb

```diff
@@ -44,12 +44,7 @@ def render_orphan_form current_orphan
      "guardian_name", "guardian_relationship",
      "guardian_id_num", "contact_number", "alt_contact_number"].each do |field|
       if orphan_full[field]
-        if !expect(rendered).to have_selector("input[id='orphan_#{field}'][value=\"#{orphan_full[field].to_s}\"]")
-          show_me_redered
-          binding.pry
```

Comment:
The `binding.pry` makes this look like it may have been a problem in the past.
However the `if` never gets evaluated because if the spec fails it bails out before the `if` is even called.